### PR TITLE
Image cleanup

### PR DESCRIPTION
**Pull request by charmander**
- [ ] Add tests, maybe

### CODE REVIEW

File: weasyl/banner.py [hunk 1]
```diff
@@ -1,8 +1,6 @@
-# banner.py
-
 from libweasyl import ratings
-
-from weasyl import media, orm
+from libweasyl import media
```
Comment (hyena) [inline weasyl/banner.py]: nit: Lexicographic ordering

File: weasyl/image.py [hunk 2]
```diff
@@ -50,74 +14,24 @@ def image_setting(im):
...
-
-    # if there's no need to resize, in-place resize is a no-op. otherwise copy
-    # the source to the destination.
-    elif not in_place:
-        files.copy(filename, destination)
-
-
-def resize_image(im, width, height):
-    return correct_image_and_call(_resize, im, width, height) or im
+        return im.original_format in ['JPEG', 'PNG', 'GIF']
```
Comment (hyena) [inline weasyl/image.py]: This part could be made a little stronger. afaict this is only used in the case of thumbnails for characters (ugh). Maybe we should not only ensure that the original_format is valid but also that the format matches the file name. Doesn't hurt to sanity check that (maybe even throw an exception if we're in a state that's permitted a file to be named something that isn't its correct type).

**Comment by hyena** [comment]
Looks good and a simpler world (+45 −198). Thanks for doing this.

+1.

File: weasyl/image.py (hunk 2 above)
Comment (charmander) [inline weasyl/image.py]: I’d like to err on the side of maintaining existing behaviour for this pull request, and just remove all the character special casing in a future one.

**Pull request merged**